integration-tests/kamelet/pom.xml

```diff
@@ -59,6 +59,10 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-bean</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-xml-io-dsl</artifactId>
```

Comment:
Can you add the `-deployment` dependency for this.

Reply:
Fixed, a `process-resources -P format -N` from root folder was missing.